src/shared/varlink-io.systemd.Machine.c

```diff
@@ -93,6 +130,10 @@ SD_VARLINK_DEFINE_INTERFACE(
                 &vl_method_Kill,
                 SD_VARLINK_SYMBOL_COMMENT("List running machines"),
                 &vl_method_List,
+                SD_VARLINK_SYMBOL_COMMENT("List images"),
+                &vl_method_ListImages,
```

Comment:
I'd prefer if we'd add a second interface for covering images. i.e. add io.systemd.MachineImage or so for this, so that the methods can then be called "List" instead of "ListImages" too.

(Sooner or later we want to cover unpriv containers nicely with machined, i.e. images i drop somewhere in $HOME. That probably means we want two instances of machined: a system wide and a per-user once. But otoh we always want to register containers on the system so that we can support hostname resolution for container names nicely, that creates this weird split that we want images per user but live machine registration for the system. Hence, as preparation for that I think it would be much nicer if we split this all up in two interfaces already to be ready for this.)

note that varlink services can implemented as many interfaces as they want on the same socket, that's entirely fine.